getting-started/macos/macos-basic-setup.md

```diff
@@ -0,0 +1,80 @@
+---
+ID:
+post_title: Basic Roots development environment for macOS
+author: Michael W. Delaney
+post_excerpt: ""
+layout: doc
+permalink: >
+
+published: false
+post_date:
+---
+
+Basic macOS 10.13+ development environment for Roots Trellis, Bedrock, and Sage
+
+## Requirements
+
+Roots Trellis, Bedrock, and Sage development on macOS is supported by several libraries and software packages. This guide will show you how to install and configure each of these recommended packages on your computer.
```

Comment:
Might be clearer to say "rely on" rather than "is supported by"?

I think it's worth including something of a disclaimer like "this guide assumes you have basic knowledge of command line and terminal usage."

Reply:
I like both of those notes. I'll make those changes.